### PR TITLE
Conda review final fixes

### PR DESCRIPTION
**Pull request by gagelarsen**

### CODE REVIEW

File: xmscore/python/misc/PyUtils.cpp [hunk 1]
```diff
@@ -168,29 +169,18 @@ py::iterable PyIterFromVecPt2d(const VecPt2d& pts)
...
-          py::tuple pt = p.cast<py::tuple>();
-          if (py::len(pt) != 3) {
-              throw py::type_error("Second arg must be an n-tuple of n-tuples of 3-tuples");
-          } else {
-              xms::Pt3d point(pt[0].cast<double>(), pt[1].cast<double>(), pt[2].cast<double>());
-              vec_pt3d.at(j) = point;
-          }
+  for (auto pts : pt3d2d) {
+      if (!py::isinstance<py::iterable>(pts)) {
+          throw py::type_error("Second arg must be an n-tuple of n-tuples of 1, 2, or 3-tuples");
```
Comment (Tsubashi) [inline xmscore/python/misc/PyUtils.cpp]: This error doesn’t match what we are checking. If the error message is correct, then we should adjust our checks accordingly.

Comment (gagelarsen) [reply xmscore/python/misc/PyUtils.cpp]: fixed

**Comment by Tsubashi** [comment]
Looks like the AppVeyor failure was caused by latency from conan-center. I’m going to merge it anyway.

**Comment by Tsubashi** [comment]
Go ahead and delete your branch if you are finished with it.

**Pull request merged**